pkg/distro/fedora/images.go

```diff
@@ -641,6 +639,82 @@ func iotSimplifiedInstallerImage(workload workload.Workload,
 	return img, nil
 }
 
+func coreosImage(workload workload.Workload,
+	t *imageType,
+	bp *blueprint.Blueprint,
+	options distro.ImageOptions,
+	_ map[string]rpmmd.PackageSet,
+	_ []container.SourceSpec,
+	rng *rand.Rand) (image.ImageKind, error) {
+
+	container, ref, err := makeOSTreePayloadContainer(options.OSTree, t.OSTreeRef())
+	if err != nil {
+		return nil, fmt.Errorf("%s: %s", t.Name(), err.Error())
+	}
+	img := image.NewOSTreeContainerDiskImage(container, ref)
+
+	distro := t.Arch().Distro()
+
+	customizations := bp.Customizations
+	img.Users = users.UsersFromBP(customizations.GetUsers())
+	img.Groups = users.GroupsFromBP(customizations.GetGroups())
+
+	img.Directories, err = blueprint.DirectoryCustomizationsToFsNodeDirectories(customizations.GetDirectories())
+	if err != nil {
+		return nil, err
+	}
+	img.Files, err = blueprint.FileCustomizationsToFsNodeFiles(customizations.GetFiles())
+	if err != nil {
+		return nil, err
+	}
+
+	img.KernelOptionsAppend = []string{
+		"rw",
+		"console=tty0",
+		"console=ttyS0",
+	}
+	img.Keyboard = "us"
```

Comment:
Does `Keyboard` and `Locale` options have any effects on the manifest?

Reply:
Adds the appropriate stages to the deployment pipeline: https://github.com/achilleas-k/images/blob/b19437c5d35c5c9028a37b14048352360288e79a/pkg/manifest/ostree_deployment.go#L361-L377

Reply:
Those should actually be part of some image specific configuration (imageconfig or default workload or something) rather than hardcoded in the image function, but that's a more general problem I'm cleaning up separately.

Reply:
Makes me wonder: Should we meddle with the default keyboard/locale while deploying the container? I think I would rather leave the defaults as defined in the container.

Reply:
Dropped.  But we should allow customizing these through the blueprint.  For all ostree images.  And the Anaconda ISO through kickstart (now that we added the commands in osbuild).

I think it's time to do the rewrite of the allowed options for image types.

Reply:
Agreed, for every image type, we should support as many customizations possible. :)